SecretSanta/src/SecretSanta.Domain/Models/Group.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Domain.Models
+{
+    public class Group : Entity
+    {
+        public Group(string title)
+        {
+            Title = title;
+            Users = new List<User>();
+        }
+
+        private string _Title;
+        public string Title
+        {
+            get => _Title;
+            set
+            {
+                if (value is null)
+                {
+                    throw new ArgumentNullException("value");
+                }
+
+                value = value.Trim();

```

Comment:
Again a stylistic critique, but this can be condensed down into the assignment below
```suggestion
                _Title = value.Trim();
```
And again you can turn the two separate checks into a single if statement